prompt-management/features/essential/version-control.mdx

```diff
@@ -0,0 +1,157 @@
+
+```typescript
+import { getPrompt, getPromptVersion, setupLangWatch } from "langwatch";
+
+// Initialize LangWatch for tracing and related features
+await setupLangWatch();
+
+// Get a specific prompt (latest version by default)
+const prompt = await getPrompt("customer-support-bot");
+
+// The prompt object contains version information
+console.log(`Prompt: ${prompt.name}`);
+console.log(`Version: ${prompt.version}`);
+console.log(`Version ID: ${prompt.versionId}`);
+
+// Get a specific version of a prompt
+const specificVersion = await getPromptVersion(
+  "customer-support-bot",
+  "version_abc123"
+);
+
+// Compile with variables
+const compiledPrompt = specificVersion.compile({
+  user_name: "John Doe",
+  user_email: "john.doe@example.com",
+  input: "I need help with my account",
+});
+```
+
+</Tab>
+<Tab title="REST API">
+
+Use the REST API to manage prompt versions:
+
+```bash
+# Get all versions of a prompt
+curl -X GET "https://app.langwatch.ai/api/prompts/{prompt_id}/versions" \
+  -H "Authorization: Bearer your-api-key"
+
+# Get a specific prompt (latest version)
+curl -X GET "https://app.langwatch.ai/api/prompts/{prompt_id}" \
+  -H "Authorization: Bearer your-api-key"
+
+# Create a new version
+curl -X POST "https://app.langwatch.ai/api/prompts/{prompt_id}/versions" \
+  -H "Authorization: Bearer your-api-key" \
+  -H "Content-Type: application/json" \
+  -d '{
+    "configData": {
+      "version": 2,
+      "prompt": "Updated prompt text...",
+      "messages": [...],
+      "inputs": [...],
+      "outputs": [...],
+      "model": "openai/gpt-4o-mini"
+    },
+    "commitMessage": "Improved customer support prompt"
+  }'
+```
+
+</Tab>
+</Tabs>
+
+## Version Information
+
+Each prompt version includes metadata such as version number, commit message, author, and creation timestamp. For detailed information about the version structure and data model, see the [Data Model](/prompt-management/data-model) page.
+
+## Version History
+
+LangWatch provides a clear view of how your prompts have evolved over time:
+
+<Frame>
+  <img
+    className="block"
+    src="/images/prompts/prompt-manager-edit.png"
+    alt="LangWatch Prompt Editor"
+  />
+</Frame>
+
+### Viewing Version History
+
+1. **Version List**: See all versions with timestamps and commit messages
+2. **Commit Messages**: Understand what was changed in each version
+3. **Author Information**: See who made each change
+4. **Version Selection**: Switch between different versions to view their content
+
+## Rollbacks
+
+You can easily rollback to previous versions:
+
+1. Navigate to the prompt's version history
+2. Select the version you want to restore
+3. Create a new version based on the selected version
+4. The new version will include the restored content
+
+## Team Collaboration
+
+All prompt versions are shared across your project team. For organization-scoped prompts, versions are also shared across all projects in the organization.
+
+<Frame>
```

Comment:
this is also not showing the scope selector, perhaps screenshot the handle modal taht opens